src/components/buttonMenu/buttonMenu.svelte

```diff
@@ -31,6 +31,12 @@
     if (isOpen === undefined) isOpenInternal = toggleTo
     onChange?.({ isOpen: toggleTo })
   }
+
+  const handleClose = (e: CloseEventDetail) => {
+    if (isOpen === undefined) isOpenInternal = false
+    onClose?.(e)
+    onChange?.({ isOpen })
```

Comment:
How about forcing the event value to `false` here?
    onChange?.({ isOpen: false })
```